docs/npwd/start/Config.md

```diff
@@ -5,11 +5,11 @@ sidebar_label: Advanced Configuration
 ---
 
 NPWD includes a `config.json` file in the root directory that allows for further customization of phone
-behavior. This file is what allows for us to offer framework-agnostic compatibility as well as for multitude
+behaviour. This file is what allows for us to offer framework-agnostic compatibility as well as for multitude
```

Comment:
This is also already spelled correctly.

Reply:
Since this is a american/british English thing, I'll simply keep as was.

Reply:
Can we change color to colour